### DISCUSSION TITLE
Use newer WebView2 version with SDK

### DISCUSSION THREAD

**Discussion by kegesch** [microsoft/WindowsAppSDK#3380] [Q&A]
We are developing an App with .NET MAUI which latest version uses the `WindowsAppSDK` 1.2.221116.1. It seems that `WindowsAppSDK` bundles `WebView2` instead of referencing the nuget version. 
Our App needs to access the handler `CoreWebView2.CoreWebView2ServerCertificateErrorDetected` which is available since WebView2 version 1.0.1245.22. However the latest `WindowsAppSDK`'s version of WebView2 does not contain that handler. It is unclear which version WebView2 is exactly used and why it is apparently such an old version. 

Is there a way to use a newer WebView2 version?

**Comment by christopher-carmichael** [comment]
I also need this. Anyone have a workaround?

**Comment by zadjii** [comment]
I think you're ultimately going to run into this:
https://github.com/microsoft/microsoft-ui-xaml/issues/5689

Looks like WindowsAppSDK doesn't declare WebView2 as a package dependency, in bundles it directly into the WASDK package. So trying to use two versions SxS is neigh impossible. I'd recommend following/bumping that thread for upddates.
